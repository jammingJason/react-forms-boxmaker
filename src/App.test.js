import { render, screen } from '@testing-library/react';
import App from './App';

test('renders App', () => {
  render(<App />);
  const linkElement = screen.getByText('X');
  expect(linkElement).toBeInTheDocument();
});

test('matches snapshot', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
