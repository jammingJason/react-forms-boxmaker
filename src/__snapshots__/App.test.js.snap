// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="App Box-div"
      id="0"
      style="text-align: right; background: rgb(232, 207, 207); position: absolute; left: 45%; top: 250px; height: 150px; width: 150px;"
    >
      <button
        class="Box-btn"
      >
        X
      </button>
    </div>
    <hr />
    <form>
      <hr />
      <h1>
        Create a New Box!
      </h1>
      <label
        for="height"
      >
        Box Height: 
      </label>
      <input
        id="height"
        name="height"
        type="text"
        value=""
      />
      <br />
      <label
        for="width"
      >
        Box Width: 
      </label>
      <input
        id="width"
        name="width"
        type="text"
        value=""
      />
      <br />
      <label
        for="color"
      >
        Color: 
      </label>
      <input
        id="color"
        name="color"
        type="color"
        value=""
      />
      <br />
      <button>
        Create Box
      </button>
    </form>
  </div>
</DocumentFragment>
`;
